### PR TITLE
Fix handling of overridden paths from database

### PR DESCRIPTION
**Pull request by rhatdan**
If the first time you run podman in a user account you do a
su - USER, and the second time, you run as the logged in USER
podman fails, because it is not handling the tmpdir definition
in the database. This PR fixes this problem.

vendor containers/common v0.11.1

This should fix a couple of issues we have seen in podman 1.9.1
with handling of libpod.conf.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@mheon @giuseppe @vrothberg @baude @QiWang19 @jwhonce @TomSweeneyRedHat PTAL

File: cmd/podman/login.go [hunk 1]
```diff
@@ -46,7 +46,6 @@ func init() {
 
 	// Podman flags.
 	flags.BoolVarP(&loginOptions.tlsVerify, "tls-verify", "", false, "Require HTTPS and verify certificates when contacting registries")
-	flags.BoolVarP(&loginOptions.GetLoginSet, "get-login", "", false, "Return the current login user for the registry")
```
Comment (mheon) [inline cmd/podman/login.go:49]: Why are we dropping this? Seems unrelated to the rest of the PR

**Comment by mheon** [comment]
One question otherwise LGTM

File: cmd/podman/login.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply cmd/podman/login.go:49]: Ditto @mheon's question.  If we are dropping it, we need a man page change.

Comment (rhatdan) [reply cmd/podman/login.go:49]: @mtrmac Added this option to github/container/common/pkg/auth, so it is conflicting in podman code and no longer needed.
